sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -39,7 +39,7 @@ class IncrementalExecution(
     val checkpointLocation: String,
     val runId: UUID,
     val currentBatchId: Long,
-    offsetSeqMetadata: OffsetSeqMetadata)
+    private[sql] val offsetSeqMetadata: OffsetSeqMetadata)
```

Comment:
just make it val. Anything inside sql.execution does not show up in docs, and therefore we just keep to val so that we can debug when we need to dig deep.